### PR TITLE
labgrid-powerrelay: add recipe

### PR DESCRIPTION
**Pull request by sknsean**
Rest api for control gpio chips

Signed-off-by: Sean Nyekjaer <sean.nyekjaer@prevas.dk>

### CODE REVIEW

File: recipes-devtools/python/python3-powerrelay/labgrid-powerrelay.service [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Type=simple
+Environment="LC_ALL=en_US.UTF-8"
+Environment="LANG=en_US.UTF-8"
+
+ExecStart=/bin/bash -c '/usr/bin/powerrelay run /etc/config.yaml'
+
```
Comment (krevsbech) [inline recipes-devtools/python/python3-powerrelay/labgrid-powerrelay.service:12]: Is it strictly necessary to wrap the powerrelay bin in "/bin/bash -c" ?

File: recipes-devtools/python/python3-powerrelay_git.bb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    python3-yarl \
+    python3-multidict \
+"
+
+SRC_URI = " \
+    git://github.com/prevas-dk/labgrid-powerrelay.git;protocol=https;branch=master \
+    file://labgrid-powerrelay.service \
+    "
+SRCREV = "eced6843b34e59187278120401dcbd0f4b3b3016"
+
```
Comment (krevsbech) [inline recipes-devtools/python/python3-powerrelay_git.bb]: Building this version based on the other PR's for the python dependencies I get the following run time error:
multidict<5.0,>=4.0 required. 
Installed is: 3.3.2
Just manually bumping to 4.5.2 on the target solves the issue. I would suggest to pin the multidict 4.5.2 in the powerrelay setups file.

**Review by krevsbech — CHANGES_REQUESTED** [review]

File: recipes-devtools/python/python3-powerrelay/labgrid-powerrelay.service (hunk 1 above)
Comment (jluebbe) [reply recipes-devtools/python/python3-powerrelay/labgrid-powerrelay.service:12]: I agree with @krevsbech, the `bash -c` shouldn't be here.

File: recipes-devtools/python/python3-powerrelay/labgrid-powerrelay.service [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Powerrelay init
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Type=simple
+Environment="LC_ALL=en_US.UTF-8"
+Environment="LANG=en_US.UTF-8"
```
Comment (jluebbe) [inline recipes-devtools/python/python3-powerrelay/labgrid-powerrelay.service]: You could probably use `C.UTF-8` here.

File: recipes-devtools/python/python3-powerrelay_git.bb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+SYSTEMD_SERVICE_${PN} = "labgrid-powerrelay.service"
+
+do_install_append() {
+    rm -rf "${D}${datadir}"
+    install -d ${D}${systemd_system_unitdir}
+    install -m 0644 ${WORKDIR}/labgrid-powerrelay.service ${D}${systemd_system_unitdir}/
+}
+
+FILES_${PN} += "${sysconfdir}/profile.d/localeenv.sh ${systemd_system_unitdir}"
```
Comment (jluebbe) [inline recipes-devtools/python/python3-powerrelay_git.bb]: Where does localeenv.sh come from?

**Review by jluebbe — CHANGES_REQUESTED** [review]

**Comment by sknsean** [comment]
@jluebbe comments fixed

**Review by jluebbe — APPROVED** [review]

**Pull request merged**